### ISSUE TITLE
Flicker when navigating in the patcher

### ISSUE THREAD

**Issue by FarmYard-Gaming** [Golem642/CtrRGBPAT2#2]
https://github.com/Golem642/CtrRGBPAT2/assets/64561006/78f550e9-4069-4013-826e-924bad9c006c

It's most noticeable when you scroll up and down. However it also happens when you press anything and the screen updates - A, B etc. On my 2DS it only seems to affect a diagonal area for some reason, which kind of shows on video.

**Comment by Golem642** [comment]
Yeah, that's because everytime you do an action it refreshes the text display (basically cleans the screen and print out everything again).
It doesn't do it in some cases for example when you click "Test LED" multiple times, that's because the screen doesn't refresh and you can see that because the logs keeps adding up at the bottom line
Maybe i'll make a dynamic refresh to only the part it needs to at some point, but it's not planned for now.
Also might as well make an actual UI instead of fixing this idk, might try to do something

**Comment by FarmYard-Gaming** [comment]
That's fair enough, thanks for the explanation!
